### DISCUSSION TITLE
(Pre)releases

### DISCUSSION THREAD

**Discussion by ThatOneCalculator** [hyprwm/Hyprland#111] [General]
When can we expect to see release numbers, even if they're prealpha/alpha/beta? No rush, just curious :)

**Comment by vaxerski** [comment]
never.








jk. Idk, soon maybe.

**Comment by vaxerski** [comment]
done

**Reply by ThatOneCalculator** [reply]
I'll add it to the AUR asap!

**Comment by ThatOneCalculator** [comment]
@vaxerski Can you maybe version it as "v0.1.0"?

```
$ makepkg --printsrcinfo > .SRCINFO
==> ERROR: pkgver is not allowed to contain colons, forward slashes, hyphens or whitespace.
```

**Reply by vaxerski** [reply]
can't you just `sed -s "s/\-//g"`?

**Reply by ThatOneCalculator** [reply]
Technically yes, but it "0.1.0-beta" doesn't follow semantic versioning and is just kinda clunky to work with :/

**Reply by vaxerski** [reply]
as well as 0.1.0 is not 0.1.0 beta

**Reply by ThatOneCalculator** [reply]
`v0.1.0.beta` would work

**Reply by vaxerski** [reply]
mfg ok

**Reply by Mrfiregem** [reply]
Hyphens are usually replaced with an underscore according to the Arch wiki: https://wiki.archlinux.org/title/PKGBUILD#pkgver

**Reply by ThatOneCalculator** [reply]
ah oki ty

**Reply by ThatOneCalculator** [reply]
@vaxerski If you're gonna use that version can you rename the tar accordingly too :pray: 
![image](https://user-images.githubusercontent.com/44733677/170778300-f40e9430-5786-457b-be1e-4a06b8a5c93a.png)

**Reply by vaxerski** [reply]
cant have shit on this planet

**Reply by vaxerski** [reply]
done

**Reply by ThatOneCalculator** [reply]
Done! Now both `hyprland-git` and `hyprland` are on the AUR.
